spec/lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation_spec.rb

```diff
@@ -474,6 +474,520 @@
     end
   end
 
+  context("secondary step by steps") do
```

Comment:
These are great! Perhaps one more that tests what happens when there is an active step by step too?

Reply:
done :)